requirements.txt

```diff
@@ -1 +1,3 @@
-
+requests==2.31.0
+bs4==0.0.2
```

Comment:
.

Reply:
??

Reply:
do you think this is a proper package that you need to install?